low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -163,12 +165,13 @@ def initialize(self, conf_reader, msgQlist, product):
         self._epoch_time = str(int(time.time()))
         self._raid_drives = []
         self._raid_device = "N/A"
-        self.os_sensor_type = {
+        self.sensor_data = {

```

Comment:
Can we chnage "self.sensor_data" ==> "self.node_sensor_data" ?

Reply:
By the naming of this module 'node_data_msg_hanlder', this `sensor_data` is basically `node sensor data` only. So I kept it shortly. As per the resource map, resource type is not differentiated like node:os, node:sw, node:hw. Now all server resources are falling under one category (node:hw).

Reply:
Okay